.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+# Simple Impalib Build Stage GitHub Workflow
+# Version: 0.1.0
+
+name: Impalib Build
+
+on: push
+
+jobs:
+  build:
+    name: Impalib Build Job
+    runs-on: ${{ matrix.os }}
+    strategy: 
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
@adamwcaliola @RustomAlexios 

Had a brief second thought on getting this ci going. It looked as if there was a limit to free use of the runners on GitHub actions, however it appears that if we stick to using the standard runners then we should be in the clear. 
E.g. Using `ubuntu-latest` and `macos-latest`

Otherwise we might need to revisit how we want to run the CI if at all 

Let me know if either of you have more insight on this.